test/maverick/api_test.exs

```diff
@@ -65,46 +71,22 @@ defmodule Maverick.ApiTest do
       resp =
         :hackney.post(
           "#{@host}/api/v1/route1/gimme/that/data",
-          [],
+          @headers,
           %{"magic_word" => "please"} |> Jason.encode!()
         )
 
       assert 404 == resp_code(resp)
       assert resp_content_type(resp)
-      assert "Not Found" == resp_body(resp)
+      assert %{"error_code" => 404, "error_message" => "Not Found"} == resp_body(resp)
     end
 
     test "handles error tuples from internal functions" do
       body = %{num1: 25, num2: 2} |> Jason.encode!()
-      resp = :hackney.post("#{@host}/api/v1/route1/multiply", [], body)
+      resp = :hackney.post("#{@host}/api/v1/route1/multiply", @headers, body)
 
       assert 403 == resp_code(resp)
       assert resp_content_type(resp)
-      assert "illegal operation" == resp_body(resp)
-    end
-  end
-
-  describe "ssl" do
```

Comment:
can we comment these out so we know to refactor them to accommodate how Plug lets you set up a server using SSL?